src/App.js

```diff
@@ -535,8 +537,8 @@ class App extends Component {
   longPoll() {
     axios.get("https://api.coinmarketcap.com/v2/ticker/1027/")
```

Comment:
I'm aware that we have a problem with coinmarketcap's API at the moment: #146
We're in the process of fixing it however. Would you mind uncommenting price-fetching below?